### DISCUSSION TITLE
Quarto - Multi-format Publishing

### DISCUSSION THREAD

**Discussion by giscus** [quarto-dev/quarto-cli#11418] [Blog]
# Quarto - Multi-format Publishing

In Quarto 1.3, additional formats listed in HTML documents will automatically be linked in an “Other Formats” section near the top of the page.

https://quarto.org/docs/blog/posts/2023-03-15-multi-format/index.html

**Comment by acarzfr** [comment]
Wonderful

**Comment by drfurtado** [comment]
This is very helpful for readers.

**Comment by cigit-zgy** [comment]
Soooo powerful

**Comment by Dexwel** [comment]
What a great extension

**Comment by samrit2442** [comment]
How can we incorporate the reaction button and subscribe at the end of the quarto blog page just like this page?

**Reply by drfurtado** [reply]
Subscription: https://quarto.org/docs/websites/website-blog.html#subscriptions

I believe the reactions are part of the giscus feature: https://quarto.org/docs/reference/projects/websites.html#giscus

**Comment by vitormarinh0** [comment]
Is this option available for book format?
